### DISCUSSION TITLE
How to create/edit a theme?

### DISCUSSION THREAD

**Discussion by fatiheker** [mixcore/mix.core#610] [Q&A]
I couldn't find any documentation about theme development? Could you write/share some source about MixCode theme development.

**Comment by Smilefounder** [comment]
Thanks @fatiheker for using mixcore. Let me record a quick video to show you how to create a quick theme and share with you asap within this week. In the mean time, once you install mixcore cms and run in your local machine. Login with administrator > portal > themes > then from here you can edit and create new theme via the CMS itself. 

The theme is based on Asp.Net Razor MVC structure and template. That you can refer to MS docs.

**Comment by vbasky** [comment]
Any docs about uploading theme or editing the current theme including the primary button etc.

**Reply by Smilefounder** [reply]
Under Theme section, you can try download the blank or default theme to see how the theme structure should be. The upload functional is working as well.

The easiest way to develop a theme is using the Admin Theme > Code editor to edit the theme components follow Asp.NET Razor MVC structure.
